<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlideShowViewer</class>
 <widget class="QDialog" name="SlideShowViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/icons.qrc">
    <normaloff>:/icons/MainIcon.png</normaloff>:/icons/MainIcon.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QMenuBar" name="menubar">
     <widget class="QMenu" name="menuFile">
      <property name="title">
       <string>&amp;File</string>
      </property>
      <addaction name="actionClear_All"/>
      <addaction name="separator"/>
      <addaction name="actionSave"/>
      <addaction name="actionSave_All"/>
      <addaction name="separator"/>
      <addaction name="actionClose"/>
     </widget>
     <addaction name="menuFile"/>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>11</number>
     </property>
     <property name="bottomMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="LabelTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>&lt;b&gt;Title&lt;/b&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Image">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>240</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>320</width>
         <height>240</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: white</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>MOT Slideshow Viewer</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabel1">
         <property name="text">
          <string>Receiving Status:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CMultColorLED" name="LEDStatus" native="true">
         <property name="minimumSize">
          <size>
           <width>13</width>
           <height>13</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>13</width>
           <height>13</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="ButtonJumpBegin">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res/icons.qrc">
           <normaloff>:/icons/JumpBegin.png</normaloff>:/icons/JumpBegin.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ButtonStepBack">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res/icons.qrc">
           <normaloff>:/icons/StepBack.png</normaloff>:/icons/StepBack.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LabelCurPicNum">
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>0/0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ButtonStepForward">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res/icons.qrc">
           <normaloff>:/icons/StepForw.png</normaloff>:/icons/StepForw.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="ButtonJumpEnd">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res/icons.qrc">
           <normaloff>:/icons/JumpEnd.png</normaloff>:/icons/JumpEnd.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer5_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonOk">
         <property name="text">
          <string>&amp;Close</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionClear_All">
   <property name="text">
    <string>Cl&amp;ear All</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save...</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save &amp;All...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CMultColorLED</class>
   <extends>QWidget</extends>
   <header>MultColorLED.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
